src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -702,7 +702,7 @@ public virtual bool SupportsPaging(CallNode callNode, TexlBinding binding)
 
         // Returns true if function is row scoped and supports delegation.
         // Needs to be overriden by functions (For example, IsBlank) which are not server delegatable themselves but can become one when scoped inside a delegatable function.
-        public virtual bool IsRowScopedServerDelegatable(CallNode callNode, TexlBinding binding, OperationCapabilityMetadata metadata)
+        public virtual bool IsRowScopedServerDelegatable(CallNode callNode, TexlBinding binding, OperationCapabilityMetadata metadata, bool generateHints = true)
```

Comment:
>generateHints [](http://example.com/codeflow?start=140&length=13)

Where do we call with generateHints = False?

Reply:
It will only be called from functions that are in PowerApps today